### PR TITLE
DEVDOCS-2354: Update Widget Tutorial

### PR DESCRIPTION
**Pull request by ektravel**
# [DEVDOCS-2354](https://jira.bigcommerce.com/browse/DEVDOCS-2354)

## What changed?
* updated code samples
* updated text
* updated resources

### CODE REVIEW

**Comment by uwikaiddi** [comment]
@ektravel, are all the files in this PR supposed to be here?

File: docs/api-docs/storefront/widgets/widgets-tutorial.md [hunk 1]
```diff
@@ -35,36 +39,35 @@ In `templates/pages/category.html`, add `{{{region name="category_header_banner"
...
 Host: api.bigcommerce.com
 X-Auth-Token: {{ACCESS_TOKEN}}
-X-Auth-Client: {{CLIENT_ID}}
 Accept: application/json
 ```
 
 [![Open in Request Runner](https://storage.googleapis.com/bigcommerce-production-dev-center/images/Open-Request-Runner.svg)](https://developer.bigcommerce.com/api-reference/storefront/widgets-api/regions/getcontentregions#requestrunner)
 
-Then, look for the region's name in the response.
+Look for the region’s name in the response.
```
Comment (traci707) [inline docs/api-docs/storefront/widgets/widgets-tutorial.md]: Look for the region's name in the response.

File: docs/api-docs/storefront/widgets/widgets-tutorial.md [hunk 2]
```diff
@@ -143,13 +137,13 @@ Accept: application/json
 
 ## Create a widget
 
-Now, use the widget template to create the widget. To do so, send a `POST` request to [`/v3/content/widgets`](https://developer.bigcommerce.com/api-reference/storefront/widgets-api/widget/createwidget).
+To create a widget, you will need to reference the widget template `uuid` from the previous step. Send a `POST` request to [`/v3/content/widgets`](https://developer.bigcommerce.com/api-reference/storefront/widgets-api/widget/createwidget) making sure to replace the placeholder value inside `widget_template_uuid` with your template's `uuid`.
```
Comment (traci707) [inline docs/api-docs/storefront/widgets/widgets-tutorial.md]: "To create a widget, use the widget template uuid from the previous step. Send a `POST` request to [`/v3/content/widgets`](https://developer.bigcommerce.com/api-reference/storefront/widgets-api/widget/createwidget) making sure to replace the placeholder value inside `widget_template_uuid` with your template's `uuid`.

File: docs/api-docs/storefront/widgets/widgets-tutorial.md [hunk 3]
```diff
@@ -188,21 +182,22 @@ Accept: application/json
 <div class="HubBlock-content">
 
 > ### Note
-> * Note the widget's UUID in the response for creating the placement in the next step.
+> * Make note of the widget’s `uuid` in the response. You will use it to create a placement in the next step.
```
Comment (traci707) [inline docs/api-docs/storefront/widgets/widgets-tutorial.md]: > * Make a note of the widget’s `uuid` in the response. You will use it to create a placement in the next step.

File: docs/api-docs/storefront/widgets/widgets-tutorial.md [hunk 4]
```diff
@@ -188,21 +182,22 @@ Accept: application/json
...
 
 </div>
 </div>
 </div>
 
 ## Create a placement
 
-In the UI, users drag and drop widgets to *place* them in a region on a page. Here, we'll use the API to programmatically place our widget. To do so, send a `POST` request to [`/v3/content/placements`](https://developer.bigcommerce.com/api-reference/storefront/widgets-api/placement/createplacement).
+In the control panel UI, users can drag and drop widgets to place them in a region on a page. For the purpose of this tutorial, we will use the Widgets API to place the widget programmatically.
```
Comment (traci707) [inline docs/api-docs/storefront/widgets/widgets-tutorial.md:195]: In the control panel UI, users can drag and drop widgets to place them in a region on a page. For this tutorial, we will use the Widgets API to place the widget programmatically.

File: docs/api-docs/storefront/widgets/widgets-tutorial.md [hunk 5]
```diff
@@ -227,46 +222,75 @@ Accept: application/json
...
 [![Open in Request Runner](https://storage.googleapis.com/bigcommerce-production-dev-center/images/Open-Request-Runner.svg)](https://developer.bigcommerce.com/api-reference/storefront/widgets-api/widget-template/createwidgettemplate#requestrunner)
 
-For more information, see [Page Builder Overview](https://developer.bigcommerce.com/stencil-docs/page-builder/page-builder-overview).
+To learn more about Page Builder, see [Page Builder Overview](https://developer.bigcommerce.com/stencil-docs/page-builder/page-builder-overview).
 
 ## Resources
 
 ### Related endpoints
-* [Widgets API](/api-reference/storefront/widgets-api)
+* [Get All Categories](https://developer.bigcommerce.com/api-reference/store-management/catalog/category/getcategories)
```
Comment (traci707) [inline docs/api-docs/storefront/widgets/widgets-tutorial.md]: Does Get All Categories belong here?

File: docs/api-docs/getting-started/best-practices.md [hunk 6]
```diff
@@ -1,8 +1,8 @@
-# API Best Practices
+# Best Practices
 
 <div class="otp" id="no-index">
 
-### On this page
+### On This Page
```
Comment (traci707) [inline docs/api-docs/getting-started/best-practices.md:5]: On this page

**Review by traci707 — COMMENTED** [review]
Looks good, it appears that you made all necessary changes.
However there are other files in this PR and I am not sure why.

**Pull request closed without merging**